crypto/bls/attestation_test.go

```diff
@@ -0,0 +1,41 @@
+package bls
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBlsKeyCreation(t *testing.T) {
+	var tests = map[string]struct {
+		keyPath  string
+		password string
+		wantErr  bool
+	}{
+		"valid bls key save": {
+			keyPath:  "./operator_keys/test.bls.key.json",
+			password: "test",
+			wantErr:  false,
+		},
+	}
+
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			t.Cleanup(func() {
+				dir := filepath.Dir(tt.keyPath)
+				_ = os.RemoveAll(dir)
```

Comment:
is ./operator_keys a path used for anything other than this test? If not this will delete potential user data.

Reply:
this is just for test - not used by any other tests.